ballerina/tests/csv_io.bal

```diff
@@ -99,6 +113,129 @@ isolated function testReadCsv() returns Error? {
     check csvChannel.close();
 }
 
+@test:Config {}
+isolated function testReadCsvRecord() returns Error? {
+    string filePath = RESOURCES_BASE_PATH + "datafiles/io/records/sample5.csv";
+    string[][] input = check fileReadCsv(filePath);
+    string[][] expected = [["User1", "WSO2", "10000.50"],["User2", "WSO2", "20000.50"],["User3", "WSO2", "30000.00"]];
+    test:assertEquals(input[0][0], "User1");
+    test:assertEquals(input[0][2], " 10000.50");
+    Employee[] input2 = check fileReadCsv(filePath);
+    test:assertEquals(input2[0].id, "User1");
+    test:assertEquals(input2[0].salary, 10000.50f);
+}
+
+@test:Config {}
+function testReadFileCsvAsStreamUsingResourceFile() returns error? {
+    string filePath = RESOURCES_BASE_PATH + "datafiles/io/records/sample5.csv";
+    string[][] expected = [["User1", "WSO2", "10000.50"],["User2", "WSO2", "20000.50"],["User3", "WSO2", "30000.00"]];
+    stream<string[], Error?> result = check fileReadCsvAsStream(filePath);
+    int i = 0;
+    check result.forEach(function(string[] val) {
```

Comment:
This case, it is better to have a fori loop, since the index is needed for the logic

Reply:
updated in [cfc1a26](https://github.com/ballerina-platform/module-ballerina-io/pull/664/commits/cfc1a26c5dceb963e890f8c4b24f89caff5ec23f)